iamy/awsdiff.go

```diff
@@ -434,10 +434,16 @@ func (a *awsSyncCmdGenerator) updateUsers() {
 
 		} else {
 			// Create user
-			a.cmds.Add("aws", "iam", "create-user",
-				"--user-name", toUser.Name,
-				"--path", path(toUser.Path),
-				"--tags", mapTagsToString(toUser.Tags))
+			if len(mapTagsToString(toUser.Tags)) == 0 {
```

Comment:
It seems wasteful to call `mapTagsToString` twice
```suggestion
			if len(toUser.Tags) == 0 {
```